### PR TITLE
Cleanup pch file

### PR DESCRIPTION
**Pull request by kean**
I've been reviewing the project setup and noticed an unusually large .pch file and a bridging header, so I decided to remove some of the unused imports from them. I don't think it has any particular benefits other than keeping things small and simple: in Objective-C, you want to keep the imports to a minimum.

To test:

## Regression Notes
1. Potential unintended areas of impact: n/a
2. What I did to test those areas of impact (or what existing automated tests I relied on): n/a
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:

- [ ] I have completed the Regression Notes.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

UI Changes testing checklist:
- [ ] Portrait and landscape orientations.
- [ ] Light and dark modes.
- [ ] Fonts: Larger, smaller and bold text.
- [ ] High contrast.
- [ ] VoiceOver.
- [ ] Languages with large words or with letters/accents not frequently used in English.
- [ ] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [ ] iPhone and iPad. 
- [ ] Multi-tasking: Split view and Slide over. (iPad)

### CODE REVIEW

File: WordPress/Classes/Services/LocalCoreDataService.m [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#import <CoreData/CoreData.h>
```
Comment (mokagio) [inline WordPress/Classes/Services/LocalCoreDataService.m]: Would `@import CoreData;` have worked here?

**Review by mokagio — APPROVED** [review]
Thank you for this cleanup.

File: WordPress/Classes/Services/LocalCoreDataService.m (hunk 1 above)
Comment (kean) [reply WordPress/Classes/Services/LocalCoreDataService.m]: I replaced these import with `@import` in all of the changed files from the PR.

Comment (mokagio) [reply WordPress/Classes/Services/LocalCoreDataService.m]: Thank you for considering my suggestion 🙇‍♂️

**Pull request merged**